_sass/_layout.scss

```diff
@@ -55,21 +51,20 @@
         text-decoration: none;
         padding: 5px 15px 5px 15px;
         margin-left: 1px;
-        color: white;
+        color: $scheme-color-1;
```

Comment:
> I wonder whether this and the changes below are really necessary: We could just keep the current colors for the menu so that the site does not become "all white".

I agree that this changes makes the site "too white" but my design/CSS skills aren't good enough to find a solution that works well.

I don't know how to add a colored bar at the bottom of the `header`.

![he](https://cloud.githubusercontent.com/assets/1506457/9380839/2d928556-4709-11e5-9a33-95edd9039458.jpg)

And because the `wrap` I can't only use the `background-color` of the `nav`.

![wrap](https://cloud.githubusercontent.com/assets/1506457/9380844/37b3a59c-4709-11e5-8bf6-ad5c62ae1125.jpg)

The "all white" was the best solution that I could think of.